src/shared-components/banner/style.js

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+import {
+  COLORS,
+  MEDIA_QUERIES,
+  BOX_SHADOWS,
+  SPACING,
+  TYPOGRAPHY_CONSTANTS,
+} from '../../constants';
+
+const successBannerStyles = css`
```

Comment:
i think this is from the alerts styles. Can we refactor this a bit to be more DRY since we can reuse this "success" theme with the background color, font color, etc.

Reply:
Thinking of doing a separate PR to DRY out the color styles between the <Alert />, <Banner /> and <Chip /> since they are using the same groupings of background, border and text colors.

Reply:
made new Pivotal task to track this DRY change after the emotion stuff goes in so that there's less churn around the `<Banner /> <Alert /> <Chip />` stuff.